pytorch_lightning/trainer/optimizers.py

```diff
@@ -40,6 +40,14 @@ def init_optimizers(self, model: Optional["pl.LightningModule"]) -> Tuple[List,
             )
             optim_conf = _MockOptimizer()
 
+        optimizers, lr_schedulers, optimizer_frequencies = self.configure_optimizers_and_schedulers(
```

Comment:
Can this wait until https://github.com/PyTorchLightning/pytorch-lightning/pull/9666 lands? To avoid the author getting conflicts

Reply:
yeah sure!